src/test/scala/cromwell/ArrayOfArrayCoercionSpec.scala

```diff
@@ -0,0 +1,25 @@
+package cromwell
+
+import akka.testkit._
+import cromwell.binding.types.{WdlArrayType, WdlStringType}
+import cromwell.binding.values.{WdlArray, WdlString}
+import cromwell.util.SampleWdl
+
+import scala.language.postfixOps
+
+class ArrayOfArrayCoercionSpec extends CromwellTestkitSpec("ArrayOfArrayCoercionSpec") {
+  "A task which contains a parameter " should {
+    "accept an array for the value" in {
```

Comment:
Maybe I'm missing something, but the name of the assertion here doesn't seem consistent with the name of the class?  Could some more comments clear this up?  :smile: